types/navigation/DocsNavItem.ts

```diff
@@ -0,0 +1,15 @@
+export interface DocsNavItemProps {
```

Comment:
For props declaration, you can declare them in the component file itself. No need to create these interfaces in `types` folder.